### ISSUE TITLE
Why don't we have colors in `.coafile` configuration file

### ISSUE THREAD

**Issue by khanchi97** [coala/coala#5148]

**Comment by Makman2** [comment]
What colors?

**Comment by khanchi97** [comment]
I mean something like this:
[![Screenshot_from_2018-02-13_16-01-16.png](https://s17.postimg.org/purupm4en/Screenshot_from_2018-02-13_16-01-16.png)](https://postimg.org/image/jh2rmczij/)

**Comment by Makman2** [comment]
Ah syntax highlighting. Such features are editor dependent. As I see you use vim, you want to have this for vim?

This could probably go into an own repository.

**Comment by khanchi97** [comment]
>This could probably go into an own repository?

Sorry? 😅

**Comment by Makman2** [comment]
I mean the syntax highlighting support for vim can be placed into a separate repository.

**Comment by khanchi97** [comment]
So I think https://github.com/coala/coala-vim/ this is the right place for this, right?

**Comment by Makman2** [comment]
Not necessarily, this one is for integrating coala's code analysis features. Syntax highlighting for coafiles is something different.

**Comment by khanchi97** [comment]
So can you create a new repository for this?

**Comment by Makman2** [comment]
If you like to work on it, what do you think about creating a repository on your own, and work on a separate branch (not on master). Once you have a solid prototype, we are transferring ownership of the repo to the coala org? :)

**Comment by khanchi97** [comment]
Oh yeah, it's cool.I would like to work on this :)
